python/ql/src/semmle/python/frameworks/Django.qll

```diff
@@ -1549,8 +1741,21 @@ private module Django {
     Parameter getRequestParam() { result = this.getArg(this.getRequestParamIndex()) }
   }
 
+  /** A data-flow node that sets up a route on a server, using the django framework. */
   abstract private class DjangoRouteSetup extends HTTP::Server::RouteSetup::Range, DataFlow::CfgNode {
-    abstract override DjangoRouteHandler getARouteHandler();
+    /** Gets the data-flow node that is used as the argument for the view handler. */
+    abstract DataFlow::Node getViewArg();
```

Comment:
Since all implementations currently look the same, why not put it here as the default? New classes could still override it if necessary.

Reply:
I considered doing this as well! 

I thought it was more important to be consistent with `getUrlPatternArg`. Possibly more importantly, keeping it this way, it's more clear from just looking at the subclass what is going on.

Given that it's a single line of QL code, I think the pros are bigger than the cons.